master/reference/cni-plugin/configuration.md

```diff
@@ -335,3 +335,36 @@ There are two annotations to request a specific IP address:
 > - You can only specify one IPv4/IPv6 or one IPv4 and one IPv6 address with these annotations.
 > - When `ipAddrs` or `ipAddrsNoIpam` is used with `ipv4pools` or `ipv6pools`, `ipAddrs` / `ipAddrsNoIpam` take priority.
 {: .alert .alert-info}
+
+#### Requesting a floating IP
+
+You can request a floating IP address for a pod through [Kubernetes annotations](https://kubernetes.io/docs/user-guide/annotations/) with {{site.prodname}}.
+
+- `cni.projectcalico.org/floatingIPs`: A list of floating IPs which will be assigned to the pod's workload endpoint.
+
+  Example:
+
+   ```yaml
+   annotations:
+        "cni.projectcalico.org/floatingIPs": "[\"10.0.0.1\"]"
```

Comment:
The quoting and escaping here looks surprisingly complex for YAML.  Are you sure that it is correct?

Reply:
Yeah, the value here is a `json` structure represented as a yaml string.

We could probably do something like this instead, though using mixed quotes: `'["10.0.0.1"]'`

Reply:
I'm happy as is.